wordpress.org/public_html/wp-content/plugins/plugin-directory/email/class-plugin-approved.php

```diff
@@ -29,43 +29,48 @@ function body() {
 To answer some common questions:
 
 * You must use SVN to upload your code -- we are unable to do that for you
-* Your SVN username is %2$s and your password is the same as you use to log in to WordPress.org
-* Your username is case sensitive
+* Your SVN username is `%2$s` and is case sensitive.
+* Generate your SVN password in your [WordPress.org profile](%4$s).
 * SVN will not accept your email address as a username
 * Due to the size of the directory, it may take 72 hours before all search results are properly updated
 
 To help you get started, here are some links:
 
-Using Subversion with the WordPress Plugin Directory:
+Using Subversion with the WordPress Plugin Directory:<br>
 https://developer.wordpress.org/plugins/wordpress-org/how-to-use-subversion/
 
-FAQ about the WordPress Plugin Directory:
+Generating your SVN Password:<br>
+https://make.wordpress.org/meta/handbook/tutorials-guides/svn-access/
+
+FAQ about the WordPress Plugin Directory:<br>
 https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/
 
-WordPress Plugin Directory readme.txt standard:
+WordPress Plugin Directory readme.txt standard:<br>
 https://wordpress.org/plugins/developers/#readme
 
-A readme.txt validator:
+A readme.txt validator:<br>
 https://wordpress.org/plugins/developers/readme-validator/
 
-Plugin Assets (header images, etc):
+Plugin Assets (header images, etc):<br>
 https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/
 
-WordPress Plugin Directory Guidelines:
+WordPress Plugin Directory Guidelines:<br>
```

Comment:
I'd add Plugin Check as a tool for Developers to remind to keep using it.

```
Plugin Check: <br>
https://wordpress.org/plugins/plugin-check/
```

Reply:
I don't think it hurts to add it, but I don't believe that a developer would consider it useful though. Once they get through the review, they are thinking "release" over anything else.

Reply:
Yes, but It's the tool for developers and it's practical for new releases to keep the plugin up to date with security measures.

Reply:
Fair enough. Can we make that change in a separate PR?